### PR TITLE
Create dynamic blocks

### PR DESCRIPTION
**Pull request by SantosGuillamot**
This Pull Request aims to add some dynamic blocks to better understand how it is the developer experience.

### CODE REVIEW

File: src/blocks/dynamic-non-interactive-parent/block.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		"postId",
+		"postType",
+		"queryId"
+	],
+	"attributes": {
+		"title": {
+			"type": "string",
+			"source": "text",
+			"selector": "h2",
+			"public": true,
```
Comment (luisherranz) [inline src/blocks/dynamic-non-interactive-parent/block.json]: These `public` fields are not necessary with Alpine, are they?

**Review by luisherranz — APPROVED** [review]
I don't know if we are supposed to review this, but it looks good to me. Great work, Mario 🙂

File: src/blocks/dynamic-non-interactive-parent/block.json (hunk 1 above)
Comment (SantosGuillamot) [reply src/blocks/dynamic-non-interactive-parent/block.json]: You're right! Copy & Pasting issues 😄 I've just changed it

**Comment by SantosGuillamot** [comment]
> I don't know if we are supposed to review this, but it looks good to me. Great work, Mario 🙂

Thanks for reviewing it 🙂

**Pull request merged**